### DISCUSSION TITLE
Welcome to Skybolt Discussions!

### DISCUSSION THREAD

**Discussion by matthew-reid** [Piraxus/Skybolt#44] [Announcements]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by lminkov** [comment]
Thank you for this nice system. I would like to understand its design. So, some design doc. would be helpful. It can describe the main components of the system, their purpose, how they interact with each other, etc...

**Reply by matthew-reid** [reply]
Hi @lminkov,

Yes some documentation would be very useful. I'm writing some at the moment which I plan to release in the coming weeks, along with a simplified build process, conan 2 support, and improved python integration.

I'd like to get a better idea of the potential user base for Skybolt. Are there any particular use cases or problems that you would find a geospatial simulation/visualization/analysis tool like Skybolt useful for? What application features would you be interested in?
